sklearn/ensemble/iforest.py

```diff
@@ -64,15 +64,18 @@ class IsolationForest(BaseBagging, OutlierMixin):
         If max_samples is larger than the number of samples provided,
         all samples will be used for all trees (no sampling).
 
-    contamination : float in (0., 0.5), optional (default=0.1)
+    contamination : 'auto' or float, optional (default='auto')
         The amount of contamination of the data set, i.e. the proportion
         of outliers in the data set. Used when fitting to define the threshold
-        on the decision function. If 'auto', the decision function threshold is
-        determined as in the original paper.
+        on the scores of the samples.
 
-        .. versionchanged:: 0.20
-           The default value of ``contamination`` will change from 0.1 in 0.20
-           to ``'auto'`` in 0.22.
+        - if 'auto', the threshold is determined as in the
+          original paper,
+        - if a float, the contamination should be in the range [0, 0.5].
```

Comment:
```suggestion
        - if float, the contamination should be in the range [0, 0.5].
```

Reply:
nitpick: looking at the docs the bullet points aren't indented as the other lists:

https://57939-843222-gh.circle-artifacts.com/0/doc/modules/generated/sklearn.ensemble.IsolationForest.html